renderer/src/lib/station-config.tsx

```diff
@@ -59,3 +47,32 @@ export async function restartToUpdate (): Promise<void> {
 export function openReleaseNotes (): void {
   return window.electron.openReleaseNotes()
 }
+
+export async function getDestinationWalletAddress (): Promise<string | undefined> {
+  return await window.electron.stationConfig.getDestinationWalletAddress()
+}
+
+export async function setDestinationWalletAddress (address: string | undefined): Promise<void> {
+  return await window.electron.saturnNode.setFilAddress(address)
+  // return await window.electron.stationConfig.setDestinationWalletAddress(address)
```

Comment:
is this a todo comment?

Reply:
Addressed [here](https://github.com/filecoin-station/filecoin-station/pull/334/files#diff-19b4370d78ca324abce6966ff16f5f4543e19d00b372a506de483efd59947e03)